app/base/file_handler.py

```diff
@@ -17,40 +21,57 @@ def create_images_folder(username):
     return property_img_folder
 
 
-def property_image_handler(username, image_files, save_to_folder):
+def save_images_to_temp_folder(image_files):
     """
-    Handles the docs uploaded from the web form. The docs are down sized using
-    the Pillow image library and saved to the file system. The image filenames are checked
-    using the werkzeug utilities, then the filename is saved to the list of filenames in the
-    dictionary object.
+    Generates a temporary folder for storing image files of property.
     """
-    images_list = [save_to_folder]
+    rand_dir_name = secrets.token_hex(16)
```

Comment:
I would rather use a timestamp like `tobe_processed_10:35:02_4_march_2020`  for the temp dir name maybe for the actual  image folder name as well

that helps in maintainance to know when the files were processed just by looking  at  the dir name
and of course  it will be unique always for as long as we add the seconds to the time the dir was created

Reply:
Yeah, that makes sense. I actually forgot about it since we discussed it before.

Reply:
I have used `uuid` for a temp dir and `uuid`, `date`, and `time` for the property dir. It looks something like this `property_acd87a2b-ee5f_05-March-2021_08-03-53`